src/main/scala/rx/lang/scala/ExperimentalAPIs.scala

```diff
@@ -50,7 +50,7 @@ class ExperimentalObservable[+T](private val o: Observable[T]) {
    * @return an [[Observable]] that will block the producer thread if the source emits items faster than its [[Observer]] can consume them
    * @see <a href="http://reactivex.io/documentation/operators/backpressure.html">ReactiveX operators documentation: backpressure operators</a>
    */
-  @deprecated("This is kept here only for backward compatibility.", "0.25.0")
+  @deprecated("Use [[[Observable.onBackpressureBlock(maxQueueLength:Int)*]]] instead. This is kept here only for backward compatibility.", "0.25.0")
```

Comment:
What's the meaning of `*`? And why do you sometimes use double `[[` and sometimes triple `[[[`?